src/components/common/Map.js

```diff
@@ -1,28 +1,31 @@
 import React from 'react';
 import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
-import { bool, object } from 'prop-types';
-import { geolocated } from 'react-geolocated';
+import { bool } from 'prop-types';
 
 import marker from '../../img/marker.svg';
 
 const Map = withScriptjs(withGoogleMap((props) => {
-  const { latitude, longitude } = props.coords;
+  let latitude;
+  let longitude;
+  navigator.geolocation.getCurrentPosition((position) => {
+    ({ latitude, longitude } = position.coords);
```

Comment:
You don't need parenthesis here. You can check this out to see the right use of curly braces and parenthesis in react: https://medium.com/@leannezhang/curly-braces-versus-parenthesis-in-reactjs-4d3ffd33128f